### DISCUSSION TITLE
Examples for query key factory might need an update

### DISCUSSION THREAD

**Discussion by AnnaYuS** [TanStack/query#5254] [Q&A]
In this example https://tanstack.com/query/v4/docs/vue/community/lukemorales-query-key-factory#fine-grained-declaration-by-features
```
 // Invalidate and refetch
      queryClient.invalidateQueries({ queryKey: queryKeys.todos.completed })
```
we should probably do `queryKeys.todos.completed.queryKey` or `...queryKeys.todos.completed` according to the examples in the query key factory repo: https://github.com/lukemorales/query-key-factory#fine-grained-declaration-colocated-by-features 

I'm new to tanstack query, so might be missing something 🙏

**Comment by TkDodo** [comment]
@lukemorales FYI

**Reply by lukemorales** [reply]
Thanks for tagging me @TkDodo, will open a PR updating it

**Reply by eminaliyevx** [reply]
?

**Comment by ayarayarovich** [comment]
@lukemorales hello, could you provide an example please 🙏
